sktime/annotation/adapters/_pyod.py

```diff
@@ -104,11 +101,8 @@ def _predict(self, X):
         elif labels == "indicator":
             Y_val_np = Y_np
 
-        if fmt == "dense":
```

Comment:
Removal of `_fmt` does not introduce regression because it was previously hardcoded to "sparse" and could not be modified by the user, rendering the "dense" branch effectively unreachable.